azure-pipelines.yml

```
@@ -907,6 +895,59 @@ jobs:
+    )
+
+  dependsOn: 
+  - Check_Code_Style
+
+  pool:
+    vmImage: 'windows-latest'
+
+  variables:
+    TargetBoard: ST_NUCLEO144_F767ZI
+    TargetPlatform: 'stm32'
+    BuildOptions: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)